docs/admin/install/operator/configuring-serving-cr.md

```diff
@@ -1,52 +1,42 @@
----
-title: "Configuring the Serving Operator Custom Resource"
-weight: 20
-type: "docs"
-aliases:
-- /docs/operator/configuring-serving-cr/
----
+# Configuring the Knative Serving Operator custom resource
 
-# Configuring the Serving Operator Custom Resource
+The Knative Serving Operator can be configured with the following options:
 
-The Knative Serving operator can be configured with these options:
-
-- [Version Configuration](#version-configuration)
-- [Serving Configuration by ConfigMap](#serving-configuration-by-configmap)
+- [Version configuration](#version-configuration)
+- [Knative Serving configuration by ConfigMap](#knative-serving-configuration-by-configmap)
 - [Private repository and private secret](#private-repository-and-private-secrets)
 - [SSL certificate for controller](#ssl-certificate-for-controller)
 - [Knative ingress gateway](#configuration-of-knative-ingress-gateway)
 - [Cluster local gateway](#configuration-of-cluster-local-gateway)
 - [High availability](#high-availability)
-- [System Resource Settings](#system-resource-settings)
+- [System resource settings](#system-resource-settings)
 - [Override system deployments](#override-system-deployments)
 
-## Version Configuration
+## Version configuration
 
-Cluster administrators can install a specific version of Knative Serving by using the `spec.version` field. For example,
-if you want to install Knative Serving 0.16.0, you can apply the following `KnativeServing` custom resource:
+Cluster administrators can install a specific version of Knative Serving by using the `spec.version` field.
 
-```
+For example, if you want to install Knative Serving 0.23.0, you can apply the following `KnativeServing` custom resource:
+
+```yaml
 apiVersion: operator.knative.dev/v1alpha1
 kind: KnativeServing
 metadata:
   name: knative-serving
   namespace: knative-serving
 spec:
-  version: 0.16.0
+  version: 0.23.0
 ```
 
-If `spec.version` is not specified, the Knative Operator will install the latest available version of Knative Serving.
-If users specify an invalid or unavailable version, the Knative Operator will do nothing. The Knative Operator always
-includes the latest 3 minor release versions. For example, if the current version of the Knative Operator is 0.16.x, the
-earliest version of Knative Serving available through the Operator is 0.14.0.
+If `spec.version` is not specified, the Knative Operator installs the latest available version of Knative Serving. If users specify an invalid or unavailable version, the Knative Operator will do nothing. The Knative Operator always includes the latest 3 minor release versions. For example, if the current version of the Knative Operator is v0.24.0, the earliest version of Knative Serving available through the Operator is v0.22.0.
```

Comment:
```suggestion
If `spec.version` is not specified, the Knative Operator installs the latest available version of Knative Serving. If users specify an invalid or unavailable version, the Knative Operator does nothing. 

The Knative Operator always includes the latest 3 minor release versions. For example, if the current version of the Knative Operator is v0.24.0, the earliest version of Knative Serving available through the Operator is v0.22.0.

Reply:
lol thought about doing this but wasn't sure whether to split it. Thanks.